Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -744,7 +744,16 @@
                             .map(relatedItSystemUsage => Helpers.RenderFieldsHelper.renderInternalReference(`kendo-system-usage-link`, "it-system.usage.main", relatedItSystemUsage.ItSystemUsageId, relatedItSystemUsage.ItSystemUsageName))
                             .reduce((combined: string, next: string, __) => combined.length === 0 ? next : `${combined}, ${next}`, ""))
                         .withSourceValueEchoExcelOutput()
-                        .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.systemRelations)));
+                        .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.systemRelations)))
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("Note")
+                        .withTitle("Note")
+                        .withId("note")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                        .withContentOverflow()
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderString(dataItem.Note))
+                        .withSourceValueEchoExcelOutput());
```

Comment:
no need to specify echo here since it will reuse the "rendering" unless otherwise defined and I think you want to reuse the rendering since, that one changes null to ""